### PR TITLE
PMM-7 add pmm 2.31.0 to dev

### PR DESCRIPTION
**Pull request by beatahandzelova**

### CODE REVIEW

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| api-tests | passed |

**Review by tplavcic — DISMISSED** [review]

File: sources/pmm.2.31.0.pmm-server.json [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+              "image_hash": "e9ed11994cef3f7ab33e126484d5d5991cccc00b54d066183d1c7abe8e29b802",
+              "status": "available",
+              "critical": false
+            }
+          }
+        }
+      }
+    ]
+  }
+
```
Comment (JiriCtvrtka) [inline sources/pmm.2.31.0.pmm-server.json:51]: Add empty line to the end of file.

**Review by JiriCtvrtka — APPROVED** [review]

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| api-tests | passed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| api-tests | passed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| api-tests | passed |

**Review by JiriCtvrtka — APPROVED** [review]

**Review by anton-bystrov — APPROVED** [review]

**Pull request merged**